### ISSUE TITLE
Create a registration and login page in html / css

### ISSUE THREAD

**Issue by IvanIsak2000** [Spaceblock/Spaceblock#3]

**Comment by mb6ockatf** [comment]
This one needs to be completed. We need a responsive design on all kinds of devices.
Probably we'll use bootstrap css framework to concentrate on webapp backend and not to bother with frontend too much

**Comment by IvanIsak2000** [comment]
> This one needs to be completed. We need a responsive design on all kinds of devices. Probably we'll use bootstrap css framework to concentrate on webapp backend and not to bother with frontend too much

Yeah, a good offer. Btw, I'm learning `bootstrap` now, and I understand a little how to create login page
